sugar_test.go

```diff
@@ -198,9 +214,10 @@ func TestSugarStructuredLogging(t *testing.T) {
 	}
 
 	// Common to all test cases.
+	err := errors.New("qux")
 	context := []interface{}{"foo", "bar"}
-	extra := []interface{}{"baz", false}
-	expectedFields := []Field{String("foo", "bar"), Bool("baz", false)}
+	extra := []interface{}{err, "baz", false}
+	expectedFields := []Field{String("foo", "bar"), Error(err), Bool("baz", false)}
```

Comment:
nit: let's combine these into a `var` block here as well

Reply:
Combined into a var block here too. Thank you!